File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cs
using System;
using System.Collections.Generic;
using System.Linq;
public class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[num_list.Length - 5];
        List<int> arr = num_list.ToList();
        arr.Sort();
        for(int i = 0; i< 5;i++)
        {
            arr.RemoveAt(0);
        }
        answer = arr.ToArray();
        return answer;
    }
}

File: 프로그래머스/0/181852. 뒤에서 5등 위로/뒤에서 5등 위로.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer(num_list);
    sort(answer.begin(), answer.end());
    for(int i = 0; i < 5; i++)
        answer.erase(answer.begin());
    return answer;
}